pkg/liqoctl/install/warning.go

```diff
@@ -0,0 +1,70 @@
+package install
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/liqotech/liqo/pkg/liqoctl/util"
+	"github.com/pterm/pterm"
+)
+
+type warner interface {
+	name() string
+	check(values map[string]interface{}) (bool, error)
+	warn() []string
+}
+
+type serviceWarner struct {
+	warnings []string
+}
+
```

Comment:
```suggestion
var _ warner = serviceWarner{}
```

if the interface is required, we should check that the object implements it